### PR TITLE
Louie add wind

### PR DESCRIPTION
**Pull request by the-louie**
This PR adds wind_speed to the forecast, also some minor typo fixes.

### CODE REVIEW

**Comment by the-louie** [comment]
Sorry for the silly PR name, forgot to change and it was created from the branch-name :)

**Comment by helto4real** [comment]
Sorry I missed this PR @the-louie

**Comment by helto4real** [comment]
image
Was running tests on your PR (have not setup any automatic for this project yet...) 

There seem to be a change in windspeed that does not match

**Comment by the-louie** [comment]
I have probably misunderstood something, but as I read it as `_get_forecast()` returns¥ a list of daily forecasts, so wind_speed gives the average over each day.

A quick check of the first day in the test data gave me `(1.9+1.5+0.8+0.4+0.4+0.5+0.5+0.3+0.3) / 9 = 0.733333`.

**Comment by helto4real** [comment]
> I have probably misunderstood something, but as I read it as `_get_forecast()` returns a list of daily forecasts, so wind_speed gives the average over each day.
> 
> A quick check of the first day in the test data gave me `(1.9+1.5+0.8+0.4+0.4+0.5+0.5+0.3+0.3) / 9 = 0.733333`.

Ahh let me check that, maybe I am the one misunderstood :). Ether way the test must be updated. I get back to you. Thank you for your contribution!

File: smhi/smhi/smhi_lib.py [hunk 1]
```diff
@@ -323,6 +326,7 @@ def _get_forecast(api_result: dict) -> List[SmhiForecast]:
         forecast._temperature_min = forecast_temp_min
         forecast._total_precipitation = total_precipitation
         forecast._mean_precipitation = total_precipitation / 24
+        forecast._wind_speed = acc_wind_speed / len(forecasts_day)
```
Comment (helto4real) [inline smhi/smhi/smhi_lib.py]: What do you think about we keep the _wind_speed as original and introduce a _mean_wind_speed instead? Then we do not break existing that expects 1-1 from SMHI api?

Comment (the-louie) [reply smhi/smhi/smhi_lib.py]: That is the sensible way to do it, so I just commited it :)

**Comment by helto4real** [comment]
Cause of changes in structure of the code due to adapting to google actions I took this code and made a new PR #5. Also added your code but also kept old functionality for backwards compability. I hope you are ok with that @the-louie. Also added tests and made them run ok. If you are up to it. Please review the #5 that it is like you intended it to be.

**Pull request closed without merging**